### DISCUSSION TITLE
Notes

### DISCUSSION THREAD

**Discussion by joseph-kendall** [CMSgov/price-transparency-guide#381] [Q&A] [answered]
Is there a notes section inside the machine-readable files to put extra information about a contract or the terms that isn't in the schema?

**Comment by Ginger228** [comment]
We are planning to put notes and disclaimers on the landing page of the MRF instead of within the INN or AA file.  Hope that helps.

**Reply by jltang-kp** [reply]
@joseph-kendall @Ginger228 we are planning on also placing global disclaimers for certain negotiated rates that are based off % of billed to state we have something like "90% of billed charges" for the negotiated rate field

We also are looking at what sort of information we need to post in alternative payment arrangements where a disclaimer is not enough. IE certain PMPM agreements as well as a medicare rate file.

**Comment by shaselton-usds** [comment, marked as answer]
@joseph-kendall You can leverage the `additional_information` attribute if necessary. 

What extra contract information are you including that isn't supported by the current schema? We would definitely like to try to continue iterating on the schema to support most contractual scenarios.
